### PR TITLE
Remove fixed press and hold delay

### PR DESCRIPTION
**Pull request by rainemak**
Includes two commits. First one sorts Qt related includes in webutils (should be done everywhere).

### CODE REVIEW

**Comment by jpetrell** [comment]
Works like a charm. :) :+1:

**Comment by rojkov** [comment]
LGTM

File: src/declarativewebutils.cpp [hunk 1]
```diff
@@ -43,6 +47,19 @@ bool testScreenDimensions(qreal pixelRatio) {
     return fmod(w, 1.0) == 0 && fmod(h, 1.0) == 0;
 }
 
+const QSettings &quickSettings()
+{
+    static const QSettings settings(QSettings::SystemScope, QStringLiteral("QtProject"), QStringLiteral("QtQuick2"));
```
Comment (matthewvogt) [inline src/declarativewebutils.cpp]: No need for 'static' here; you only access the settings once, so it won't be helpful to keep them in memory.

Comment (rainemak) [inline src/declarativewebutils.cpp]: That's right.

**Pull request merged**